File: 060_Julia/fizzbuzz.jl
# This is a comment.

function fizzbuzz(i)
    if (i % 3 == 0 && i % 5 == 0)
        return "FizzBuzz"
    elseif (i % 3 == 0)
        return "Fizz"
    elseif (i % 5 == 0)
        return "Buzz"
    else
        return i
    end
end

for i = 1:100
    println(fizzbuzz(i))
end

File: 061_Octave/fizzbuzz.m
# This is a comment.

for i=1:100
    if (mod(i, 3) == 0 && mod(i, 5) == 0)
        disp('FizzBuzz')
    elseif (mod(i, 3) == 0)
        disp('Fizz')
    elseif (mod(i, 5) == 0)
        disp('Buzz')
    else
        printf('%d\n', i)
    endif
end

